06-building-a-program-start-to-finish/checking-repository-health/output/README.md

```diff
@@ -0,0 +1,10 @@
+#### Output
+- **Using**: `Code Climate` and `Coveralls`
+- **Deliver**: a screenshot of your code coverage and GPA badges in your repo's README file
+- **With**: *independently*
+
+- **Screenshot**
+- Code Climate Badge Screenshot
+![Anu's Screenshot for Code Climate](Codeclimate_Screenshot.png)
+- Coveralls Badge Screenshot
+![Anu's Screenshot for Coveralls](Coveralls_Screenshot.png)
```

Comment:
Coverage has increased. Code climate on the queue. Yet to build as at when writing this message.